code/modules/reagents/chemistry/machinery/chem_master.dm

```diff
@@ -202,6 +202,8 @@
 					return
 				name = reject_bad_text(name)
 				while(count--)
+					if(reagents.total_volume <= 0)
```

Comment:
Perhaps an error message could be displayed to the user here?

Reply:
Sounds good. Will add that in soon.